kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/KafkaRequestHandler.java

```diff
@@ -174,6 +176,8 @@
     private final String advertisedListeners;
     private final int defaultNumPartitions;
     public final int maxReadEntriesNum;
+    public final ConcurrentHashMap<String, String> currentConnectedGroup;
```

Comment:
It's better to add comments to indicate what does the key and value string mean here.